### PR TITLE
Backend beta prep

### PR DESCRIPTION
**Pull request by chelshaw**

### CODE REVIEW

File: backend/resources/activity-schemas/motorbike.json
Comment (chelshaw) [inline backend/resources/activity-schemas/motorbike.json:1]: Why deleting this?

File: backend/src/fun_forecast_backend/activity_schema/dataclasses.py [hunk 1]
```diff
@@ -1,23 +1,39 @@
...
-    with_weather: List[int]
+class Wind:
+    min: int
+    max: int
+    ideal: int
+    units: str
+
+
+@dataclass
+class Temp:
```
Comment (chelshaw) [inline backend/src/fun_forecast_backend/activity_schema/dataclasses.py:14]: nit: Wind and Temp are currently equivalent, feels like they could be combined. Then again, not sure what I would call that dataclass 🤔

File: backend/src/fun_forecast_backend/forecast/forecast.py
Comment (chelshaw) [inline backend/src/fun_forecast_backend/forecast/forecast.py:1]: ✨ so many python tricks in here

File: backend/src/fun_forecast_backend/main.py [hunk 2]
```diff
@@ -1,21 +1,29 @@
...
-#     allow_credentials=True,
-#     allow_methods=["*"],
-#     allow_headers=["*"],
-# )
+app = FastAPI(title="Fun Forecast Backend")
 
-# logger = get_logger()
+app.add_middleware(
+    CORSMiddleware,
+    allow_origins=["http://myfunforecast.com", "http://beta.myfunforecast.com"],
```
Comment (chelshaw) [inline backend/src/fun_forecast_backend/main.py:21]:
```suggestion
    allow_origins=["https://myfunforecast.com", "https://beta.myfunforecast.com"],
```
Secure protocol only

File: backend/resources/activity-schemas/motorbike.json
Comment (shayaanjagtap) [reply backend/resources/activity-schemas/motorbike.json:1]: It was the previous format. Temp deleting so everything is uniform for now - my new PR will add it back in :)

**Review by shayaanjagtap — APPROVED** [review]

**Pull request merged**